### PR TITLE
Bug 1721056 - 404 error link to "bug writing guidelines" on top of enter_bug.cgi

### PR DESCRIPTION
**Pull request by dklawren**

### CODE REVIEW

File: extensions/BMO/template/en/default/pages/bug-writing.html.tmpl [hunk 1]
```diff
@@ -6,6 +6,323 @@
...
 
-[% PROCESS global/redirect.html.tmpl
-   url = "https://developer.mozilla.org/docs/Mozilla/QA/Bug_writing_guidelines"
+[% PROCESS global/header.html.tmpl
+   title = "Bug Writing Guidelines"
 %]
+
+<div class="message">
+  <p><strong>If you need help with Mozilla software (for example with Firefox, Seamonkey or Thunderbird),
+    use one of the available <a href="http://www.mozilla.org/support/">support options</a>.</strong>
```
Comment (globau) [inline extensions/BMO/template/en/default/pages/bug-writing.html.tmpl:15]: this should be https

File: extensions/BMO/template/en/default/pages/bug-writing.html.tmpl [hunk 2]
```diff
@@ -6,6 +6,323 @@
...
+    Do not edit this page!</p>
```
Comment (globau) [inline extensions/BMO/template/en/default/pages/bug-writing.html.tmpl:16]: This sentence should be removed.

File: extensions/BMO/template/en/default/pages/bug-writing.html.tmpl [hunk 3]
```diff
@@ -6,6 +6,323 @@
...
+   <li>If you have <a href="#Writing_precise_steps_to_reproduce">precise steps to reproduce</a>
+     — great! — you&apos;re on your way to reporting a useful [% terms.bug %] report.</li>
+   <li>If you can reproduce occasionally, but not after following specific steps, you must provide
+     <a href="#specific-types">additional information</a> for the [% terms.bug %] to be useful.</li>
+   <li>If you can&apos;t reproduce the problem, there&apos;s probably no use in reporting it, unless
+     you provide unique information about its occurrence.</li>
+  </ul>
+ </li>
+ <li>Make sure your software is up to date. Ideally, test an in-development version to see whether
+   your [% terms.bug %] has already been fixed (e.g. <a href="http://www.mozilla.com/en-US/firefox/channel/">
```
Comment (globau) [inline extensions/BMO/template/en/default/pages/bug-writing.html.tmpl:50]: https here too

File: extensions/BMO/template/en/default/pages/bug-writing.html.tmpl [hunk 4]
```diff
@@ -6,6 +6,323 @@
...
+   Firefox Beta, Aurora</a>, or bleeding-edge <a class="link-https" href="https://nightly.mozilla.org/">
```
Comment (globau) [inline extensions/BMO/template/en/default/pages/bug-writing.html.tmpl:51]: Aurora doesn't exist.

File: extensions/BMO/template/en/default/pages/bug-writing.html.tmpl [hunk 5]
```diff
@@ -6,6 +6,323 @@
...
+   Nightly</a>).</li>
+ <li>When reporting a Firefox [% terms.bug %], first check if you can reproduce the [% terms.bug %] in a
+   <a href="https://support.mozilla.org/en-US/kb/profile-manager-create-and-remove-firefox-profiles">
```
Comment (globau) [inline extensions/BMO/template/en/default/pages/bug-writing.html.tmpl:54]: https://support.mozilla.org/en-US/kb/profile-manager-create-remove-switch-firefox-profiles

File: extensions/BMO/template/en/default/pages/bug-writing.html.tmpl [hunk 6]
```diff
@@ -6,6 +6,323 @@
...
+     <li>If the [% terms.bug %] falls into one of <a href="#specific-types">specific types of [% terms.bugs %] listed in a section
+       below</a>, it may still be useful even if you can&apos;t reproduce it in a new Firefox profile.</li>
+   </ul></li>
+ <li>Open the <a class="link-https" href="https://bugzilla.mozilla.org/enter_bug.cgi?format=guided">
+   Enter a new [% terms.bug %]</a> form, which will guide you through most of the [% terms.bug %] reporting process:
+   <ul>
+     <li>Create a [% terms.Bugzilla %]  account if you don&apos;t have one already, select the product having the [% terms.bug %].</li>
+     <li>Enter a <strong>clear unique summary</strong> as <a href="#Writing_a_clear_summary">described below</a>;
+       check if the [% terms.bug %] has already been reported (if you want to be more thorough, there&apos;s an advanced guide
+       on <a href="/en-US/docs/Screening_duplicate_bugs">screening duplicate [% terms.bugs %]</a>).</li>
```
Comment (globau) [inline extensions/BMO/template/en/default/pages/bug-writing.html.tmpl:73]: broken link

File: extensions/BMO/template/en/default/pages/bug-writing.html.tmpl [hunk 7]
```diff
@@ -6,6 +6,323 @@
...
+
+<h3 id="For_specific_types_of_[% terms.bugs %]"><a name="specific-types">For specific types of [% terms.bugs %]</a></h3>
+
+<p>If you are reporting a <strong>crash [% terms.bug %]</strong>, please
+  <a href="/en-US/docs/How_to_get_a_stacktrace_for_a_bug_report">include a Breakpad ID or attach stack
+  trace</a>, and include the crash signature in the [% terms.bug %] summary as well as in the <strong>Crash
+  Signature</strong> field.</p>
+
+<p>If you are reporting a <strong>memory use or leak [% terms.bug %]</strong>, please
+  <a href="/en-US/docs/Mozilla/Performance/about:memory">attach the output of about:memory</a>.
```
Comment (globau) [inline extensions/BMO/template/en/default/pages/bug-writing.html.tmpl:165]: broken link

File: extensions/BMO/template/en/default/pages/bug-writing.html.tmpl [hunk 8]
```diff
@@ -6,6 +6,323 @@
...
+  Ideally, find steps to reproduce an increase in what is shown in about:memory (even after
+  clicking the &quot;Minimize memory usage&quot; button at the bottom). If you have trouble finding steps
+  to reproduce, try the Firefox Support page titled
+  <a href="https://support.mozilla.org/en-US/kb/firefox-uses-too-much-memory-ram">Firefox Uses Too Much
```
Comment (globau) [inline extensions/BMO/template/en/default/pages/bug-writing.html.tmpl:169]: https://support.mozilla.org/en-US/kb/firefox-uses-too-much-memory-or-cpu-resources

File: extensions/BMO/template/en/default/pages/bug-writing.html.tmpl [hunk 9]
```diff
@@ -6,6 +6,323 @@
...
+  Memory (Ram) - How to Fix</a>.</p>
+
+<p>If you are reporting a <strong>[% terms.bug %] about slowness or high CPU usage</strong>, please
+  <a href="/en-US/docs/Mozilla/Performance/Reporting_a_Performance_Problem">provide a link to the
```
Comment (globau) [inline extensions/BMO/template/en/default/pages/bug-writing.html.tmpl:173]: broken link

File: extensions/BMO/template/en/default/pages/bug-writing.html.tmpl [hunk 10]
```diff
@@ -6,6 +6,323 @@
...
+  performance profile in the [% terms.bug %]</a>.</p>
+
+<p>If you are reporting a <strong>hang</strong> (beachball on macOS or &quot;not responding&quot; on Windows),
+  please follow the instructions in <a href="/en-US/docs/How_to_Report_a_Hung_Firefox">How to Report a
```
Comment (globau) [inline extensions/BMO/template/en/default/pages/bug-writing.html.tmpl:177]: broken link

File: extensions/BMO/template/en/default/pages/bug-writing.html.tmpl [hunk 11]
```diff
@@ -6,6 +6,323 @@
...
+  Hung Firefox</a>.</p>
+
+<p>If you are reporting a [% terms.bug %] involving a <strong>Flash hang</strong>, please visit
+  <a href="https://wiki.mozilla.org/Flash/Hang_Debugging">https://wiki.mozilla.org/Flash/Hang_Debugging</a>
+  to learn how to provide useful information for developers.</p>
+
+<p>If you are reporting a <strong>[% terms.bug %] involving a specific web page</strong>, please try to make a
+  <a href="/en-US/docs/Reducing_testcases">reduced testcase</a> and attach it to the [% terms.bug %] report.
```
Comment (globau) [inline extensions/BMO/template/en/default/pages/bug-writing.html.tmpl:185]: broken link

File: extensions/BMO/template/en/default/pages/bug-writing.html.tmpl [hunk 12]
```diff
@@ -6,6 +6,323 @@
...
+  If you don&apos;t have time or expertise, please report the issue to <a href="https://webcompat.com/">
+  webcompat.com</a> instead, where our volunteers will do this for you.</p>
+
+<p>If the [% terms.bug %] was <strong>recently introduced</strong>, finding a
+  <a href="http://mozilla.github.io/mozregression/quickstart.html">regression window</a> can help identify
```
Comment (globau) [inline extensions/BMO/template/en/default/pages/bug-writing.html.tmpl:190]: https

File: extensions/BMO/template/en/default/pages/bug-writing.html.tmpl [hunk 13]
```diff
@@ -6,6 +6,323 @@
...
+  the cause of the [% terms.bug %].</p>
+
+<p><a href="http://www.chiark.greenend.org.uk/~sgtatham/bugs.html#intermittent">What if my [% terms.bug %] seems
```
Comment (globau) [inline extensions/BMO/template/en/default/pages/bug-writing.html.tmpl:193]: https

File: extensions/BMO/template/en/default/pages/bug-writing.html.tmpl [hunk 14]
```diff
@@ -6,6 +6,323 @@
...
+<table class="standard-table">
+  <tbody>
+    <tr>
+      <th>What should you include in a [% terms.bug %] report?</th>
+      <th>Example</th>
+    </tr>
+    <tr>
+      <td>
+        Indicate if the problem can be reproduced using a
+        <a href="https://support.mozilla.org/en-US/kb/profile-manager-create-and-remove-firefox-profiles">
```
Comment (globau) [inline extensions/BMO/template/en/default/pages/bug-writing.html.tmpl:209]: https://support.mozilla.org/en-US/kb/profile-manager-create-remove-switch-firefox-profiles

File: extensions/BMO/template/en/default/pages/bug-writing.html.tmpl [hunk 15]
```diff
@@ -6,6 +6,323 @@
...
+      <li><u>Additional Builds and Platforms</u>: Whether or not the [% terms.bug %] takes place on other platforms
+        (or browsers, if applicable). It should look something like this: <em>&quot;Doesn&apos;t Occur
+        On Mozilla/5.0 (Macintosh; Intel Mac OS X 10.10; rv:38.0) Gecko/20100101 Firefox/38.0 ID:20150330004006 CSet:
+        9e57e9776571&quot;</em>.</li>
+    </ul>
+  </li>
+  <li><strong>Steps to Reproduce</strong>: Minimized, easy-to-follow steps that will trigger the [% terms.bug %].
+    If they&apos;re necessary, make sure to include any special setup steps. A good example of this would
+    look like the following:<br>
+    1) View any web page. (I used the default sample page, http://www.google.com/).<br>
```
Comment (globau) [inline extensions/BMO/template/en/default/pages/bug-writing.html.tmpl:318]: https

**Review by globau — CHANGES_REQUESTED** [review]

**Comment by dklawren** [comment]
Thanks for the review. Anything that is broken markup was broken before on the original page. Will fix the issues.

**Comment by globau** [comment]
> Thanks for the review. Anything that is broken markup was broken before on the original page. Will fix the issues.

We can't pull in content without ensuring it's accurate and all links work.

**Review by globau — APPROVED** [review]

**Pull request merged**